auth/service_test.go

```diff
@@ -0,0 +1,192 @@
+package auth_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	testsupport "github.com/fabric8-services/fabric8-tenant/test"
+	"github.com/fabric8-services/fabric8-tenant/test/doubles"
+	"github.com/fabric8-services/fabric8-tenant/test/recorder"
+	goajwt "github.com/goadesign/goa/middleware/security/jwt"
+)
+
+func TestResolveUserToken(t *testing.T) {
```

Comment:
Can you improve AuthService coverage please?

Reply:
agree with @MatousJobanek , I can see that most of the code is covered [link](https://codecov.io/gh/fabric8-services/fabric8-tenant/pull/632/diff) and coverage looks good for auth_service :-) :+1:

Reply:
Maybe it was a codecov glitch but the report showed some big parts of the service not covered. Or maybe it's your latest commit with SA token tests... Anyway it's much better now!